src/admin/components/Alerts/DelayedJobsProgressAlert.tsx

```diff
@@ -12,47 +11,16 @@ type DelayedJobsProgressAlertProps = {
 };
 
 const DelayedJobsProgressAlert: FC<DelayedJobsProgressAlertProps> = ({ show, title, setIsLoadingDelayedJob }) => {
-  const [delayedJobProcessing, setDelayedJobProcessing] = useState<number>(0);
-  const [delayedJobTotal, setDalayedJobTotal] = useState<number>(0);
-  const [progressMessage, setProgressMessage] = useState<string>("Running 0 out of 0 polygons (0%)");
-
-  const store = useStore<AppStore>();
-  useEffect(() => {
-    let intervalId: any;
-    if (show) {
-      intervalId = setInterval(() => {
-        const { total_content, processed_content, progress_message } = store.getState().api;
-        setDalayedJobTotal(total_content);
-        setDelayedJobProcessing(processed_content);
-        if (progress_message != "") {
-          setProgressMessage(progress_message);
-        }
-      }, 1000);
-    }
-
-    return () => {
-      if (intervalId) {
-        setDelayedJobProcessing(0);
-        setDalayedJobTotal(0);
-        setProgressMessage("Running 0 out of 0 polygons (0%)");
-        clearInterval(intervalId);
-      }
```

Comment:
I saw this interval to poll the store pattern in a couple of places. One of the great things about redux is that it's easy to set up a selector that only re-draws the current component when its output changes. So, you both hear about store changes immediately, and only redraw when they happen.

See the definition of `useJobProgress()` in `DelayedJob.ts` to see the replacement.